emacs/ocamlformat.el

```diff
@@ -394,11 +394,23 @@ With ARG, perform this action that many times."
 (defun set-newline-and-indent ()
   (local-set-key (kbd "RET") 'custom_newline-and-indent))
 
-(add-hook 'tuareg-mode-hook 'ocamlformat-setup-indent t)
-(add-hook 'tuareg-mode-hook 'set-newline-and-indent)
-
-(add-hook 'caml-mode-hook 'ocamlformat-caml-mode-setup  t)
-(add-hook 'caml-mode-hook 'set-newline-and-indent)
+(defun ocamlformat-version ()
+  "Get the version of the installed ocamlformat."
+  (car
```

Comment:
I think `version<=` should be able to handle the likely snapshot version syntax (I don't know if you had a particular syntax in mind that it doesn't handle, see `version-to-list`).

Reply:
The unreleased versions are of the form `0.18.0-37-g587b36a0-dirty` and they are not handled by `version<=`, but that's okay to only keep what's on the left hand side of the first `-` for the comparison